client/client_test.go

```diff
@@ -484,24 +486,68 @@ func concurrentIncrements(db *client.DB, t *testing.T) {
 			// Wait until the other goroutines are running.
 			wgStart.Wait()
 
-			if err := db.Txn(func(txn *client.Txn) error {
-				txn.SetDebugName(fmt.Sprintf("test-%d", i))
-
-				// Retrieve the other key.
-				gr, err := txn.Get(readKey)
-				if err != nil {
-					return err
+			if isMultiphase {
```

Comment:
I'd prefer the repetition. The reason is that if something regresses, it is more likely that just one of the tests will fail; otherwise, both will fail and the cause will be harder to find.